### PR TITLE
pkp/pkp-lib#6054 Added Issues filter to Active submissions list panel

### PR DESCRIPTION
**Pull request by thinkbulecount2**
For this issue: https://github.com/pkp/pkp-lib/issues/6054

### CODE REVIEW

File: pages/dashboard/DashboardHandler.inc.php [hunk 1]
```diff
@@ -119,10 +119,11 @@ function index($args, $request) {
 						'param' => 'assignedTo',
 						'value' => [],
 						'filterType' => 'pkp-filter-autosuggest',
+						'component' => 'field-select-users',
 						'autosuggestProps' => [
 								'allErrors' => (object) [],
 								'apiUrl' => $request->getDispatcher()->url($request, ROUTE_API, $context->getPath(), 'users', null, null, ['roleIds' => [ROLE_ID_MANAGER, ROLE_ID_SUB_EDITOR]]),
-								'component' => 'field-autosuggest',
+								'component' => 'field-select-users',
```
Comment (NateWr) [inline pages/dashboard/DashboardHandler.inc.php:126]: I _think_ that this `component` is not necessary. The only one you need is the one at line 122 above.

File: pages/dashboard/DashboardHandler.inc.php [hunk 2]
```diff
@@ -139,6 +140,34 @@ function index($args, $request) {
...
+									'label' => __('issues.submissions.issueIds'),
+									'locales' => [],
+									'name' => 'issueIds',
+									'primaryLocale' => 'en_US',
+									'selectedLabel' => __('common.assigned'),
+									'value' => [],
+								]
+							]
+						]
+					];
```
Comment (NateWr) [inline pages/dashboard/DashboardHandler.inc.php:170]: We want the issues filter to appear on _every_ submissions list, but only under the following conditions:

1. We're in OJS, not OMP or OPS.
2. The user is an admin, manager, editor or assistant.

In this case, I think the best thing to do is to extend `SubmissionsListPanel.inc.php`, which is only in OJS. I'll leave a comment on that PR with some more details.

Comment (thinkbulecount2) [reply pages/dashboard/DashboardHandler.inc.php]: Resolved in https://github.com/pkp/pkp-lib/issues/6054

File: classes/components/listPanels/PKPSubmissionsListPanel.inc.php [hunk 3]
```diff
@@ -20,207 +20,207 @@
...
+
+        return array_merge(
+            array(
+                'contextId' => $contextId,
+                'count' => $this->count,
+                'offset' => 0,
+            ),
+            $this->getParams
+        );
+    }
```
Comment (NateWr) [inline classes/components/listPanels/PKPSubmissionsListPanel.inc.php:225]: Looks like your code editor adjusted the whitespace here from tabs to spaces.

**Pull request merged**